addons/watch/stringtable.xml

```diff
@@ -95,6 +106,7 @@
             <Spanish>Pies</Spanish>
             <Japanese>フィート</Japanese>
             <Italian>Piedi</Italian>
+            <German>Feet</German>
```

Comment:
Feet is not entirely wrong but in german the measurement is usually translated and given in "Fuß"